File: GrifballWebApp.Server/EventOrganizer/SeasonDto.cs
namespace GrifballWebApp.Server.EventOrganizer;

public class SeasonDto
{
    public int SeasonID { get; set; }
    public string SeasonName { get; set; } = null!;
    public DateTime SignupsOpen { get; set; }
    public DateTime SignupsClose { get; set; }
    public DateTime DraftStart { get; set; }
    public DateTime SeasonStart { get; set; }
    public DateTime SeasonEnd { get; set; }
    public int SignupsCount { get; set; }
    public string? CopyFrom { get; set; }
    public bool CopyAvailability { get; set; }
    public bool CopySignups { get; set; }
    public bool CopyTeams { get; set; }
}

File: grifballwebapp.client/src/app/api/dtos/seasonDto.ts
import { DateTime } from "luxon";

export class SeasonDto {
  seasonID: number = 0;
  seasonName: string = "";
  signupsOpen!: DateTime;
  signupsClose!: DateTime;
  draftStart!: DateTime;
  seasonStart!: DateTime;
  seasonEnd!: DateTime;
  signupsCount: number = 0;
  copyFrom: string | null = null;
  copyAvailability: boolean = false;
  copySignups: boolean = false;
  copyTeams: boolean = false;
}
